### ISSUE TITLE
Option to control how N/A cells are displayed

### ISSUE THREAD

**Issue by Frakenz** [yesoreyeram/yesoreyeram-boomtable-panel#61]
I would like to be able to change the text and background color of these cells.
In some panels I would like to set the background in an N/A cell to red so that I can see I am missing data in queries that have a very short time range, where fill(null) doesn't work because it doesn't detect previous values. I would also like to have the possibility to add an awesome font with some text to that cell.

Another simple use would be to leave the cell blank when there are too many N/A. Since that just adds noise to the table with no valuable information.

Thanks for the awesome plugin btw :)

**Comment by yesoreyeram** [comment]
Have u tried [v1 branch](https://github.com/yesoreyeram/yesoreyeram-boomtable-panel/tree/v1) ? That have options to handle null values gracefully. Let me know if that solve your problem or not.

![image](https://user-images.githubusercontent.com/153843/53055320-26fbee80-349f-11e9-94ec-183a4facea32.png)
![image](https://user-images.githubusercontent.com/153843/53055891-47c54380-34a1-11e9-95de-f1f19a40422a.png)
![image](https://user-images.githubusercontent.com/153843/53055900-4e53bb00-34a1-11e9-93bf-6c1b5f3dfee3.png)


Leave blank if you dont want BG color or text for null values
![image](https://user-images.githubusercontent.com/153843/53055845-249a9400-34a1-11e9-9818-49d4a5429539.png)

**Comment by yesoreyeram** [comment]
This is merged into master and part of v1.0.0 release d14e1e09a03257d0fa9008087da8714368d0d1ff

v1.0.0 provides options to control the display of N/A cells using the options tab

![image](https://user-images.githubusercontent.com/153843/53410428-b3387500-39bb-11e9-9e0b-01413f2a2396.png)

**Comment by Frakenz** [comment]
Thank you, that looks great.
I have not had the chance to try it yet, that's why I had not replied.
Do you know when we will see the new version in grafana.com?

**Comment by yesoreyeram** [comment]
Thanks @Frakenz . There is a pull request pending with grafana. https://github.com/grafana/grafana-plugin-repository/pull/387 Once that is merged, v1.0.0 will be available through grafana.com and cli.

**Comment by Frakenz** [comment]
@yesoreyeram Just tested this and it looks great. Thank you!
